relayer/chains/bitcoin/provider.go

```diff
@@ -0,0 +1,1437 @@
+package bitcoin
```

Comment:
please consider refactoring this file to multiple files(listener,routes,tx and so on) to avoid creating too large file.